akka-stream-tests/src/test/java/akka/stream/javadsl/RetryFlowTest.java

```diff
@@ -36,7 +36,7 @@ public RetryFlowTest() {
 
   public static
   // #withBackoff-signature
-  <In, InCtx, Out, OutCtx, Mat> Flow<In, Out, Mat> withBackoff(
+  <In, Out, Mat> Flow<In, Out, Mat> withBackoff(
```

Comment:
I wonder if this signature stuff adds much value? What do you think about linking to apidoc instead? as in https://github.com/akka/akka/pull/28090

Reply:
I think it is noisy and error prone to duplicate the signature for each operator so 👍 on linking to (j)API docs instead.

Reply:
A link to API docs is a good idea.
Re-implementing the Java signature has the positive effect that there is a compile time check that the signature works as expected.